lib/index.ts

```diff
@@ -15,3 +15,8 @@ export { default as Scope } from "./Scope";
 export { default as Input } from "./components/Input";
 export { default as Textarea } from "./components/Textarea";
 export { default as Select } from "./components/Select";
+
+/**
+ * Types
+ */
+export { SubmitHandler } from "./Form";
```

Comment:
Should we separate exports of types from others? Because this can be easily appended to export from Form:
```js
export { default as Form, SubmitHandler } from "./Form";
```

Reply:
it's not required, but improve the readability. when we open this file we can easily see which `types` we are exporting. it's useful, for example, to create docs about types. with only one type this may seem not necessary, but is good to keep a pattern right now and and keep it as library grows.